tsl/src/remote/connection.c

```diff
@@ -2278,8 +2278,8 @@ remote_connection_end_copy(TSConnection *conn, TSConnectionError *err)
 				 * writeable, we just retry.
 				 */
 				(void) WaitLatchOrSocket(MyLatch,
+										 WL_TIMEOUT | WL_SOCKET_WRITEABLE | WL_EXIT_ON_PM_DEATH,
```

Comment:
Is the `WL_EXIT_ON_PM_DEATH` constant actually needed or is it a leftover from the first version of the PR?

Reply:
Yes, it is required according to the comments to `WaitLatchOrSocket`.

Reply:
good catch :)